doc/hostname.md

```diff
@@ -0,0 +1,49 @@
+## Hostname in Linux ##
+
+We have several hostnames in linux. Every of them has different usage and different place where to set it up. Some of them can be configured in the YaST's lan module.
+
+## Local system hostname / static hostname (/etc/hostname) ##
+
+You can put a string into /etc/hostname. This string is than used as so called "local system hostname" which is set in the kernel during the boot. This name is used for identification of your local system. This name is typically used e.g. in logs where individual log entries can be prepended by this name. More importantly this name is not related to networking, so putting e.g. FQDN (Fully Qualified Domain Name) has no sense.
+
+Content of this file can be displayed by ```hostname``` command or in systemd world by ```hostnamectl --static```
```

Comment:
Markdown: Inline, a pair of single backquotes is enough. Triple backquotes are for multiline code
```md
    ```sh
    cat /etc/hostname
    cat /etc/HOSTNAME
    ```
```

vs
```md
    Blah `cat /etc/hostname` blah
```

Reply:
To be pedantic, which is the point of this file: No.

`hostname` displays the kernel hostname, like `uname -n` does. That kernel hostname is initialized from `/etc/hostname` (probably) by `hostnamectl`. Just running `hostnamectl` reads the file and sets it. (even if using the `status` subcomand!)

Reply:
Right. I've updated the description to be (hopefully) more precise.